services/apps/portal/src/application-review/application-review.controller.ts

```diff
@@ -0,0 +1,115 @@
+import { ServiceNotFoundException } from '@app/common/exceptions/base.exception';
+import { Mapper } from '@automapper/core';
+import { InjectMapper } from '@automapper/nestjs';
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+  Req,
+  UseGuards,
+} from '@nestjs/common';
+import { LocalGovernmentService } from '../alcs/local-government/local-government.service';
+import { APPLICATION_STATUS } from '../application/application-status/application-status.dto';
+import { ApplicationService } from '../application/application.service';
+import { AuthGuard } from '../common/authorization/auth-guard.service';
+import { User } from '../user/user.entity';
+import {
+  ApplicationReviewDto,
+  UpdateApplicationReviewDto,
+} from './application-review.dto';
+import { ApplicationReview } from './application-review.entity';
+import { ApplicationReviewService } from './application-review.service';
+
+@Controller('application-review')
+@UseGuards(AuthGuard)
+export class ApplicationReviewController {
+  constructor(
+    private applicationService: ApplicationService,
+    private applicationReviewService: ApplicationReviewService,
+    private localGovernmentService: LocalGovernmentService,
+    @InjectMapper() private mapper: Mapper,
+  ) {}
+
+  @Get('/:fileNumber')
+  async get(@Param('fileNumber') fileNumber: string, @Req() req) {
+    const userLocalGovernment = await this.getUserGovernment(req.user.entity);
+
+    const applicationReview = await this.applicationReviewService.get(
+      fileNumber,
+      userLocalGovernment,
+    );
+
+    if (!applicationReview) {
```

Comment:
It looks like this will error out on service level since it uses getOrFail

Reply:
Good catch, removed